src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,21 @@
-export const TodoList = () => {};
+import { FC } from 'react';
+import { Todo } from '../../api/todos';
+import { TodoInfo } from '../TodoInfo';
+// import { User } from '../../api/users';
+
+interface Props {
+  todos: Todo[];
+  // users: User[];
```

Comment:
The `users` array is commented out in the `Props` interface. If `TodoInfo` requires user data, consider uncommenting this line and passing the `users` array as a prop to `TodoInfo` to ensure proper data flow.